tests/core/pyspec/eth2spec/test/altair/transition/test_transition.py

```diff
@@ -23,6 +23,12 @@ def _state_transition_and_sign_block_at_slot(spec, state):
     return sign_block(spec, state, block)
 
 
+def _next_epoch_via_signed_block(spec, state):
```

Comment:
Consider adding a flag `next_epoch_via_block(spec, state, insert_state_root=True)` and then actually putting this signing version of the function (`next_epoch_via_signed_block`)  inside of `helpers/state.py`